notifier.rst

```diff
@@ -553,6 +553,26 @@ and
 :class:`Symfony\\Component\\Notifier\\Notification\\EmailNotificationInterface`
 also exists to modify messages send to those channels.
 
+Development & Debugging
+-----------------------
+
+Disabling Delivery
+~~~~~~~~~~~~~~~~~~
+
+While developing (or testing), you may want to disable delivery of notifications entirely.
+You can do this by forcing Notifier to use the ``NullTransport`` for all configured texter
+and chatter transports in only the ``dev`` environment:
```

Comment:
Since in the beginning you mention "developing (or testing)" we should also mention both environments here?
```suggestion
and chatter transports in only the ``dev`` (or respectively ``test``) environment:
```